modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/fragment/renderer/SpacesSectionFragmentRenderer.java

```diff
@@ -92,6 +111,51 @@ public void render(
 		}
 	}
 
+	private ObjectValuePair<JSONArray, Long> _getAssetLibrariesObjectValuePair(
+		ThemeDisplay themeDisplay) {
+
+		try {
+			AssetLibraryResource.Builder builder =
+				_assetLibraryResourceFactory.create();
+
+			AssetLibraryResource assetLibraryResource = builder.user(
+				themeDisplay.getUser()
+			).build();
+
+			Page<AssetLibrary> assetLibrariesPage =
```

Comment:
@adolfopa, code looks good, I just have a comment about the 5 items we retrieve. Should we order this somehow? At the moment it will be retreived based on order from the DB.

Reply:
Once we get the pin/unpin feature in place we'll need to rewrite that anyway, so right now the order doesn't really matter (as long as it displays something).

Reply:
Perfect!